app/src/main/java/com/example/a442projects_thisappslaps_co/Settings/SettingsFragment.java

```diff
@@ -1,16 +1,44 @@
 package com.example.a442projects_thisappslaps_co.Settings;
 
+import android.content.SharedPreferences;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
 import android.widget.ImageButton;
+import android.widget.CompoundButton;
 
+
+import androidx.appcompat.widget.SwitchCompat;
 import androidx.fragment.app.Fragment;
 
 import com.example.a442projects_thisappslaps_co.R;
 
 public class SettingsFragment extends Fragment implements View.OnClickListener {
+    SwitchCompat switch3;
```

Comment:
Make these member variables private, and prepend an 'm' to each of the member variables. So like "mSwitch3". This is from our in class Activity.